modules/lib/src/main/resources/assets/js/app/wizard/page/LiveEditPageProxy.ts

```diff
@@ -457,7 +465,7 @@ export class LiveEditPageProxy {
                 const event: ComponentLoadedEvent = new ComponentLoadedEvent(newComponentView, componentView);
                 event.fire(this.liveEditWindow);
 
-                newComponentView.select();
+                newComponentView.select(clickPosition, menuPosition, newlyCreated, rightClicked, avoidInspectComponentRefresh);
```

Comment:
Too many parameters. Find a way to use `ItemViewSelectedEventConfig` here, and send in only config parameters that are needed.